llvm/lib/Target/AMDGPU/AMDGPU.td

```diff
@@ -1269,6 +1269,10 @@ def FeatureDynamicVGPRBlockSize32 : SubtargetFeature<"dynamic-vgpr-block-size-32
   "Use a block size of 32 for dynamic VGPR allocation (default is 16)"
 >;
 
+def FeatureLshlAddU64
```

Comment:
I wasn't sure whether to include "-inst" in the name.

Reply:
We already have something like `FeatureAtomicFMinFMaxF64FlatInsts` right?

Reply:
OK, yes, the convention seems to be to use `Inst` or `Insts` in the name.